base/compiler/ssair/passes.jl

```diff
@@ -1231,7 +1231,7 @@ function sroa_mutables!(ir::IRCode, defuses::IdDict{Int, Tuple{SPCSet, SSADefUse
             else
                 phiblocks = iterated_dominance_frontier(ir.cfg, ldu, get(lazydomtree))
             end
-            allblocks = sort!(vcat(phiblocks, ldu.def_bbs); alg=QuickSort)
+            allblocks = vcat(phiblocks, ldu.def_bbs)
```

Comment:
It looks like the only query we need is `in`, so it should perhaps be a (Bit)Set, which probably wasn't available at the time this code was initially written
```suggestion
            allblocks = union!(Set(phiblocks), ldu.def_bbs)
```

Reply:
Seems reasonable; we'll have to see if it is practically an improvement to use a Set b.c. Vectors are better than Sets for `in` on very small collections.